apps/tom-server/docker.md

```
@@ -0,0 +1,79 @@
+
+The image ships the bundled translations into `/usr/share/twake/chat/tom/i18n/`. To override with custom locale files:
+
+```sh
+-v /host/path/to/i18n/:/usr/local/share/twake/chat/tom/i18n/:ro
+```
+
+### Landing page (static)
+
+The server landing page is a single HTML file. Default path:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)